scipy/linalg/tests/test_decomp.py

```diff
@@ -1251,6 +1251,50 @@ def test_simple(self):
                                   [[1, 0, 0], [0, 0, 0], [0, 0, 0]])
 
 
+class TestHOSVD:
+    def setup_method(self):
+        seed(1234)
+
+    def test_simple(self):
+        A_exact = np.array([[[0.5, 1, 3], [2, 4, 1.7]],
+                            [[3.5, -3, 0.8], [0.3, -23, 337]],
+                            [[4.4, 31.23, 11.223], [1, 2, 3]]])
+
+        for full_tensor in (True, False):
+            U, S = hosvd(A_exact, full_tensor=full_tensor)
+
+            A = S
+            for i in range(len(U)):
+                A = np.tensordot(A, U[i], (0, 1))
+
+            assert_array_almost_equal(A, A_exact)
```

Comment:
Deprecated, please use `assert_allclose` here and below.
```suggestion
            assert_allclose(A, A_exact)
```